builder/docker/artifact_import.go

```diff
@@ -29,7 +30,20 @@ func (a *ImportArtifact) Id() string {
 }
 
 func (a *ImportArtifact) String() string {
-	return fmt.Sprintf("Imported Docker image: %s", a.Id())
+	tags := a.StateData["docker_tags"]
+	if tags == nil {
+		return fmt.Sprintf("Imported Docker image: %s", a.Id())
+	} else {
+		cast := tags.([]interface{})
+		names := []string{}
+		for _, name := range cast {
+			if n, ok := name.(string); ok {
+				names = append(names, n)
+			}
+		}
+		return fmt.Sprintf("Imported Docker image: %s with tags %s",
+			a.Id(), strings.Join(names, " "))
+	}
```

Comment:
NIT: the first if could be used as a guard clause
```suggestion
	}
	cast := tags.([]interface{})
	names := []string{}
	for _, name := range cast {
		if n, ok := name.(string); ok {
			names = append(names, n)
		}
	}
	return fmt.Sprintf("Imported Docker image: %s with tags %s",
		a.Id(), strings.Join(names, " "))
```